app/models/flood_risk_engine/enrollment.rb

```diff
@@ -20,6 +19,8 @@ class Enrollment < ActiveRecord::Base
     # The Correspondence Contact Details related to this Application a.k.a Main Contact
     belongs_to :correspondence_contact, class_name: "Contact"
 
+    has_one :location, as: :locatable, dependent: :restrict_with_exception
```

Comment:
ok. I just now we did exactly what you are doing in WEX, with site address on Enrollment and then had to change it later on 

We know bridges, and other structures likely to come along plus already have     `has_many :enrollment_exemptions` and `FloodRiskEngine::EnrollmentExemption`  .. so seems pretty easy to delegate activity_location to  the EnrollmentExemption .. but :ship: